lib/alephant/broker/load_strategy/s3/sequenced.rb

```diff
@@ -15,7 +15,7 @@ def s3_path(component_meta)
               component_meta.options,
               sequence(component_meta)
             ).tap do |obj|
-              fail InvalidCacheKey if obj.location.nil?
```

Comment:
You'll need to double check any applications consuming this gem in case they're relying on this custom exception